README.md

```diff
@@ -0,0 +1,17 @@
+# Building-with-Backgrounds-and-Gradients
+
+This is my solo project on the Microverse Main Curriculum 
+list. 
+
+## Project goals 
+* Is to learn how to use background images gradients.
+* To learn leaner gradients
+
+## Screenshot
+![alt text](https://imagensn.com/images/2019/09/05/Screenshot-from-2019-09-05-23-26-31-min.png "Screenshot")
```

Comment:
Try to use use the full-screen mode of your explorer to take the screenshot, that way your readme file will look more professional (and avoid having the zoom bar showing as well)